### ISSUE TITLE
data download links are broken

### ISSUE THREAD

**Issue by samarth-robo** [facebookresearch/ContactPose#27]
I am working for a solution to this, will update this issue for public visibility.

**Comment by samarth-robo** [comment]
Most of the data is now available in IEEE DataPort as https://dx.doi.org/10.21227/fb0w-gt48.

TODO
- [ ] Upload `images_full` and share
- [ ] Update code to use this new data source

**Comment by rohitdavas** [comment]
did you get around it ?

**Comment by taeyeopl** [comment]
@samarth-robo Does it mean subscribing to IEEE DataPort to download?

**Comment by rohitdavas** [comment]
Hi, I just checked that the IEEE Dataport subscription has a cost. If it's possible to have the dropbox API working, it will be helpful.

Also, creating a single zip of 600 GB is hard to download :(

**Comment by rohitdavas** [comment]
Is it possible to get the small sample dataset to work with in the meantime ?

**Comment by samarth-robo** [comment]
> @samarth-robo Does it mean subscribing to IEEE DataPort to download?

Yes, for now. I am planning to make ContactPose available as a huggingface dataset, but it is happening at a slow pace. @taeyeopl 

> Is it possible to get the small sample dataset to work with in the meantime ?

Yes, that's a great idea! I can upload that to Google Drive and share the link. Will do it for the sample data that is mentioned in the readme and share the link here.

**Comment by rohitdavas** [comment]
> > @samarth-robo Does it mean subscribing to IEEE DataPort to download?
> 
> Yes, for now. I am planning to make ContactPose available as a huggingface dataset, but it is happening at a slow pace. @taeyeopl
> 
> > Is it possible to get the small sample dataset to work with in the meantime ?
> 
> Yes, that's a great idea! I can upload that to Google Drive and share the link. Will do it for the sample data that is mentioned in the readme and share the link here @rohitdavas

Thanks. 

That would be really helpful. 

HF-dataset. Nice !

**Comment by matteo-carbonari** [comment]
> > @samarth-robo Does it mean subscribing to IEEE DataPort to download?
> 
> Yes, for now. I am planning to make ContactPose available as a huggingface dataset, but it is happening at a slow pace. @taeyeopl
> 
> > Is it possible to get the small sample dataset to work with in the meantime ?
> 
> Yes, that's a great idea! I can upload that to Google Drive and share the link. Will do it for the sample data that is mentioned in the readme and share the link here @rohitdavas

Hello! Are there any updates regarding the dataset? 

Thank you, 

Matteo

**Comment by samarth-robo** [comment]
@rohitdavas @senakilinc @matteo-carbonari @ywang1121 sorry for the delay.

Here is a [Google Drive link](https://drive.google.com/file/d/1paUAxXgHp6wDFBFw9MI1mxGElEl2KPew/view?usp=share_link) to sample data.

Instructions (assume `CONTACTPOSE_ROOT` denotes the ContactPose repository directory):
1. Delete or rename the `CONTACTPOSE_ROOT/data`.
2. Download this ZIP file to `CONTACTPOSE_ROOT`.
3. Unzip this file. It will create `CONTACTPOSE_ROOT/ContactPose sample data`. Rename this directory to `CONTACTPOSE_ROOT/data`.

I am working on the HF dataset. Trying to figure out how to include the object meshes; everything else seems straightforward.

**Comment by starfaerie** [comment]
Any updates here on the HF release? Been almost 7 months

**Comment by Tobio-c** [comment]
Any updates here ？
